Dockerfile

```diff
@@ -0,0 +1,31 @@
+# This Dockerfile allows you to run python.cz website easily and without
+# issues with dependencies - locally, on your server or in cloud like now.sh.
+# That's handy for testing or for demonstration of new feature in a pull request.
+#
+# For running the website locally:
+#
+#     docker build -t python.cz .
+#     docker run -p 8000:8000 python.cz
+#
+# ...and open in your browser: http://localhost:8000
+#
+# Environment variables can be specified too>
+#
+#     docker run -p 8000:8000 -e GITHUB_TOKEN=token123 python.cz
+
+FROM python:3.4
```

Comment:
Nechceme to valit na alpine? Trochu bikeshading, ale IMHO jsou ty obrazy menší a víc self-contained.

Reply:
Proč ne, když se tu na systém ani nesahá (žádné apt-get install), upraveno,